src/torch_models.py

```diff
@@ -302,10 +302,14 @@ def classify(self, x: Array) -> bool:
 class MLPClassifier(Classifier, nn.Module):
     """MLPClassifier definition."""
 
-    def __init__(self, in_size: int, max_itr: int) -> None:
+    def __init__(self, in_size: int, max_itr: int, seed: int = None) -> None:
```

Comment:
I'm surprised mypy doesn't complain -- shouldn't this type be `Optional[int]`?

Reply:
what was the resolution?

Reply:
changed it to `Optional[int]`